src/main/java/com/salesforce/dataloader/process/ProcessRunner.java

```diff
@@ -131,6 +138,9 @@ public synchronized void run(ILoaderProgress monitor) {
             // load parameter overrides (from command line or caller context)
             logger.info(Messages.getString("Process.loadingParameters")); //$NON-NLS-1$
             config.loadParameterOverrides(getConfigOverrideMap());
+            
+            // Make sure that the required properties are specified.
+            validateConfigProperties(config);
```

Comment:
This method throws ProcessInitializationException which has useful info, so it might be better to handle it similar to what we do for ApiFault below, where we get the exception message from that Exception otherwise this info will be lost.

Reply:
I do not follow this point. The catch block on line 184 picks up the exception message when creating an instance of RuntimeException:
` catch (Exception e) {
            throw new RuntimeException(e);
        }`

Reply:
I am proceeding to merge the PR.  I can fix issues, in case there are any that come up post-merge.